src/middleware/v1/registration/death.js

```diff
@@ -66,7 +67,7 @@ module.exports = {
         audit.create(ri.username, ri.client, req.url, ri.groups, 'search', 'death')
           .then(() => model.search(query))
           .then(r => {
-            res.send(r);
+            res.send(redactDeath(r, ri.roles));
```

Comment:
We should map `redactDeath` over `r`.

```
res.send(r.map(e => redactDeath(e, ri.roles)));
```

Reply:
Or something like that.

Reply:
We should also put the existing version in to the `read` function earlier in the file. I forgot to mention that.